views/layouts/layout-pane.njk

```diff
@@ -23,7 +23,7 @@
       <div class="govuk-prose-scope">
         {{ contents | safe }}
       </div>
-
+      {% include "_contact-panel.njk" %}
```

Comment:
do we want the contact panel on all pages? or only on the styles/components/patterns pages as in the prototype?

Reply:
You're right, I've fixed this by wrapping it inside `{% if section %}`